dcap/types/src/quote3.rs

```diff
@@ -208,32 +233,39 @@ impl TryFrom<Vec<u8>> for Quote3<Vec<u8>> {
 /// Table 8 of
 /// <https://download.01.org/intel-sgx/latest/dcap-latest/linux/docs/Intel_SGX_ECDSA_QuoteLibReference_DCAP_API.pdf>.
 #[derive(Debug, Clone, Hash, PartialEq, Eq)]
-struct AuthenticationData<'a> {
-    bytes: &'a [u8],
-    // Since this has to be read always we unpack for availability.
-    // This is the `size` field, *not* the length of `bytes`.
-    data_size: usize,
+struct AuthenticationData<T> {
```

Comment:
What's the goal behind switching this to a type param?

Reply:
Reverted back to lifetime on a slice